wordpress-mobile/WordPress-iOS
# ULS: Remove unifiedKeychainLogin flag.

Auth PR: https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/479

This removes the `unifiedKeychainLogin` feature flag. In Auth, the master `unifiedAuth` flag is used to show keychain login.

To test:
- Run the app. Logout if necessary.
- Verify the keychain login appears on the Prologue view.

PR submission checklist:
